scripts/lp.js

```diff
@@ -0,0 +1,27 @@
+
+$("#block-yui_3_17_2_1_1530853676327_8650").appendTo(".su-form-tradie");
+$("#block-yui_3_17_2_1_1530857589866_35655").appendTo(".su-form-host");
+
+  $('.su-exit').click(function() {
+    $('#signup-opacity').removeClass('suo-active');
+    $('#signup-outer').removeClass('su-active');
+  })
+
+	$('.lp-signup-open').click(function() {
+		$('#signup-opacity').addClass('suo-active');
+		$('#signup-outer').addClass('su-active');
+	})
+
+  $('.su-toggle-host').click(function() {
+    $('.su-toggle-host').removeClass('su-toggle-not-active');
+    $('.su-toggle-tradie').addClass('su-toggle-not-active');
```

Comment:
why don't we delegate all this to the body and we may be able to clean up a lot of these selectors

my gut tells me that we have redundant information in the DOM after all these operations